includes/insights-session-health-mobile.mdx

```diff
@@ -0,0 +1,51 @@
+## Charts
+
+![Insights Mobile Session Health Tab](./common-imgs/img/mobile-insights-session-health.png)
+
+For the Mobile module, the following charts are available:
+
+- [Crash Free Sessions](/product/releases/health/#crash-free-sessionsusers)
+- [Total Sessions by Release](/product/releases/health/#session)
+- Release Adoption
+- Session Health
+- Session Counts
+- User Health
+- User Counts
+- [New Issues by Release](/product/issues/)
+
+Most of these charts, with the exception of the New Issues by Release chart, are based on session-backed data.
+
+The Session Health, Session Counts, User Health, and User Counts charts aim to break down trends in how your sessions are performing status-wise. The status options are healthy, crashed, errored, or abnormal. [Learn more about session statuses](/product/releases/health/#session-status).
+
+The Crash Free Sessions chart extracts out a single status — "crashed" — to highlight the crash free rate of the 5 most adopted releases in that project. In this case, we define "adoption" as the percent of sessions occurring in that release, out of the total number of sessions in that project. We chose to highlight crash free rate, since this is typically the most useful and relevant error-related status for mobile developers.
+
+The Total Sessions by Release and Release Adoption charts are two ways of visualizing the same data — how many sessions occur in each release. The most recent 5 releases are shown. Total Sessions by Releases is count-based, and Release Adoption is percentage-based (the percent of session counts coming from that release).
+
+The New Issues by Release chart is the only chart that does not rely on session-backed data. Instead, it shows the number of new issues per release over time. The most recent 5 releases are shown, and there is an additional series labeled `other` that comprises all other releases. (This series is turned off in the legend by default.) It also gives a preview of the two most recently seen issues for the selected projects. The chart header, when hovered, contains a link to view all issues for the selected projects.
+
+![Issues chart](./common-imgs/img/mobile-insights-issue-chart.png)
+
+Click on any option in the graph legend to hide it. In some charts, including the Session Health chart, the healthy series is hidden by default to allow you to automatically see the errored, abnormal, and crashed series in better scale.
+
+The overall chart view can be customized, so that you can display the visualizations you care about most. Simply click the toggle near the chart title to see other charts that can be displayed.
```

Comment:
```suggestion
The overall chart view can be customized, so that you can display the visualizations and metrics you care about most. Simply click the toggle near the chart title to see other charts that can be displayed.
```